### PR TITLE
#1843: Media Gallery grid is scrolled down when text doesn't pass the validation on Edit Image page

### LINKED ISSUE #30165: Media Gallery grid is scrolled down when text doesn't pass the validation on Edit Image page
### Preconditions (*)
1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Several images added to Media Gallery

### Steps to reproduce (*)
1. Go to **Content - Media Gallery**, select the first image from the grid
2. Click "three dots" and select **Edit**
3. Type not valid symbol into _New Tag_ field, asterisk (*) for example  
4. Click **Save**
5. Observe the behavior of the Media Gallery, behind the _Edit Image_ window

### Expected result (*)
 The Media Gallery is not scrolled down

### Actual result (*)
The Media Gallery is scrolled down after each click on **Save** button. When _Edit Image_ window is closed, user is not in the same part of the grid as previously
edit_save_validation

### PR DESCRIPTION
**Pull request by yolouiese**
### Description (*)
This PR fixes the scrolling feature of the page when the Media Gallery edit modal is opened and the validation script is triggered.

### Fixed Issues (if relevant)
1. Fixes https://github.com/magento/magento2/issues/30165 : Media Gallery grid is scrolled down when text doesn't pass the validation on Edit Image page

### Manual testing scenarios (*)
**Preconditions**
1. Install Magento with Adobe Stock Integration
2. Configured integration in Stores -> Configuration -> Advanced-> System -> Adobe Stock Integration fieldset
3. Several images added to Media Gallery

**Steps to reproduce**
1. Go to Content - Media Gallery, select the first image from the grid
2. Click "three dots" and select Edit
3. Type not valid symbol into New Tag field, asterisk (*) for example
4. Click Save
5. Observe the behavior of the Media Gallery, behind the Edit Image window

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by yolouiese** [comment]
@magento run all tests

**Comment by yolouiese** [comment]
@magento run all tests

File: app/code/Magento/MediaGalleryUi/view/adminhtml/requirejs-config.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+var config = {
+    config: {
+        mixins: {
+            'mage/validation': {
+                'Magento_MediaGalleryUi/js/lib/mage/validation-mixin': true
```
Comment (sivaschenko) [inline app/code/Magento/MediaGalleryUi/view/adminhtml/requirejs-config.js]: Can we avoid mixin creation and fix the issue either in `mage/validator` directory or adjusting validation usage on Media Gallery side

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the PR @yolouiese ! Please see my comment

**Comment by yolouiese** [comment]
@magento run all tests

**Comment by sidolov** [comment]
@yolouiese  I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**